cohortextractor2/query_language.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
There was some discussion about whether this was a bad idea or not;  I'm not sure we came to any conclusions, but maybe it should be discussed some more